docs/source/checkpoints/intro.rst

```diff
@@ -43,7 +43,7 @@ The following example shows the contents of a quantized model intended to be ser
     └── tokenizer_config.yaml
 
 Community Checkpoint Converter
------------------------------
+------------------------------
```

Comment:
Edits for Line 7. We like to use imperative verbs for a procedure title, rather than a gerund. suggested revision:

Change "Understanding Checkpoint Formats" to "Understand Checkpoint Formats."

Reply:
How about just "Checkpoint Formats"?

Reply:
That's great.